dev-libs/rapidjson/rapidjson-1.1.0.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A fast JSON parser and generator for C++ with both SAX/DOM style API"
+DESCRIPTION="RapidJSON is a JSON parser and generator for C++. \
+It was inspired by RapidXml. RapidJSON is small but complete. \
+RapidJSON is fast. Its performance can be comparable to strlen().\
+RapidJSON is self-contained and header-only. RapidJSON is memory-friendly. \
+RapidJSON is Unicode-friendly"
+HOMEPAGE="https://github.com/Tencent/rapidjson"
+COPYRIGHT="2015 THL A29 Limited, a Tencent company
+		2015 Milo Yip"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz"
+CHECKSUM_SHA256="bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e"
+SOURCE_FILENAME="rapidjson-1.1.0.tar.gz"
+SOURCE_DIR="rapidjson-1.1.0"
+PATCHES="rapidjson-1.1.0.patchset"
```

Comment:
`SOURCE_DIR` will always default to recipe name (sans `.recipe` part), so you won't need it here

```diff
-SOURCE_FILENAME="rapidjson-1.1.0.tar.gz"
-SOURCE_DIR="rapidjson-1.1.0"
-PATCHES="rapidjson-1.1.0.patchset"
+SOURCE_FILENAME="rapidjson-$portVersion.tar.gz"
+PATCHES="rapidjson-$portVersion.patchset"